### ISSUE TITLE
b-form-tags binding not work in Vue 3

### ISSUE THREAD

**Issue by EagleTsui** [bootstrap-vue/bootstrap-vue#7093]
I upgrading my Vue 2 project to Vue 3 (BootstrapVue with @vue/compat).  I cannot bind data correctly when using "b-form-tags".  I setup a [Demo](https://stackblitz.com/edit/bootstrap-vue-with-compat-kvodtu?file=vite.config.js,_gitignore,index.html,components%2FHelloBootstrap.vue).  In this demo contains 2 "b-form-tags" elements, once is using v-model (**id:tags-basic**) and another one is no binding (**id:tags-basic2**), when I try to input value in (**id:tags-basic**), the array always contains the first character but another one (**id:tags-basic2**) is no problem.

**Comment by kovsu** [comment]
But there seems to be no problem in [the official website](https://bootstrap-vue.org/docs/components/form-tags).

**Comment by EagleTsui** [comment]
> But there seems to be no problem in [the official website](https://bootstrap-vue.org/docs/components/form-tags).

The official website is using Vue 2, the Vue 3 version v2.23.1 released by [xanf](https://github.com/xanf).  The Vue 2 version is no problem but Vue 3 has this issue.

**Comment by VividLemon** [comment]
Yes, this is an issue that only affects Vue 3, and would not affect the docs.

**Comment by VividLemon** [comment]
Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.

**Comment by EagleTsui** [comment]
> Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.

Is bootsrap-vue will be obsoleted?  if not that any plan to resolve this issue?

**Comment by VividLemon** [comment]
> > Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.
> 
> Is bootsrap-vue will be obsoleted? if not that any plan to resolve this issue?

Neither repository is obsolete. The only issue is, only Xanf knows how to fix these @vue/compat specific issues. So, you will likely need to wait until he has time to fix this issue.

**Comment by kovsu** [comment]
> > Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.
> 
> Is bootsrap-vue will be obsoleted? if not that any plan to resolve this issue?

Maybe you can fix it : ), if the author don't have time.

**Comment by VividLemon** [comment]
> > > Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.
> > 
> > 
> > Is bootsrap-vue will be obsoleted? if not that any plan to resolve this issue?
> 
> Maybe you can fix it : ), if the author don't have time.

By that logic you could fix it

**Comment by kovsu** [comment]
> > > > Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.
> > > 
> > > 
> > > Is bootsrap-vue will be obsoleted? if not that any plan to resolve this issue?
> > 
> > 
> > Maybe you can fix it : ), if the author don't have time.
> 
> By that logic you could fix it

If no one fixes it, I'll try it in a few days

**Comment by EagleTsui** [comment]
> > > > > Hypothetically you could use https://github.com/cdmoro/bootstrap-vue-3 and only import this component.
> > > > 
> > > > 
> > > > Is bootsrap-vue will be obsoleted? if not that any plan to resolve this issue?
> > > 
> > > 
> > > Maybe you can fix it : ), if the author don't have time.
> > 
> > 
> > By that logic you could fix it
> 
> If no one fixes it, I'll try it in a few days

Hi Kovsu, any good news?

**Comment by siah64** [comment]
:value="value"
      @input="value = $event"

**Comment by siah-chong** [comment]
I am not fully sure what is going on but if you replace 
v-model="value"
with
      :value="value"
      @input="value = $event"
from the demo. The strange behaviour seems to get fixed.
I think it has something to do with the change in behaviour of v-model from vue 2 to vue3.
https://v3-migration.vuejs.org/breaking-changes/v-model.html

**Comment by chinciusan** [comment]
I've had issues with this too. For me 
```
:value="value"
@input="value = $event"
```
worked only visually, but the value was not updated, and a workaround was creating a handler that just sets the new value for the data: 

```
:value="emails"
@input="handleTagInput"
...
handleTagInput(val) {
    this.emails = val;
},
```
